docs/api-reference/org/regenerate-project-api-key-ideally-use-when-no-api-key-is-available-as-project-can-have-multiple-api-keys.mdx

```diff
@@ -0,0 +1,3 @@
+---
+openapi: post /api/v1/org/projects/{projectId}/api-key/regenerate
```

Comment:
The filename for this endpoint is quite long and may cause issues with some file systems. Consider using a shorter, more concise name while maintaining clarity.